src/lib.rs

```diff
@@ -4,4 +4,161 @@
 //! Supports the [ESP32-WROOM-32E](https://www.espressif.com/sites/default/files/documentation/esp32-wroom-32e_esp32-wroom-32ue_datasheet_en.pdf), [ESP32-WROOM-32UE](https://www.espressif.com/sites/default/files/documentation/esp32-wroom-32e_esp32-wroom-32ue_datasheet_en.pdf) modules.
 //! Future implementations will support the [ESP32-WROOM-DA](https://www.espressif.com/sites/default/files/documentation/esp32-wroom-da_datasheet_en.pdf) module.
 //! 
-//! NOTE This crate is still under active development. This API will remain volatile until 1.0.0
\ No newline at end of file
+//! NOTE This crate is still under active development. This API will remain volatile until 1.0.0
+
+
+// This is just a placeholder for now. We will need to implement a socket struct or
+// use an existing crate
+type Socket = [u8; 1];
+
+// TODO: Find a way to share these between NinaCommandHandlers
+const START_CLIENT_TCP: u8 = 0x2du8;
+const GET_FW_VERSION: u8 = 0x37u8;
```

Comment:
You have these again below and I remember we discussed having them being more struct-specific than in the global scope. So these two might be considered superfluous.

Reply:
Still trying to figure out how to accomplish that sort of encapsulation.